spec/test-outputs/www-production.out.vcl

```diff
@@ -203,6 +203,17 @@ sub vcl_recv {
     set req.http.original-url = req.url;
   }
 
+  if (req.url.path ~ "(?i)^/register-to-vote-armed-forces") {
+    set req.url.path = "/register-to-vote-armed-forces";
```

Comment:
Do we need to set `req.url.path` if we're setting `req.url` as a whole in the next line?

Reply:
I commented on this in the `vcl_templates/www.vcl.erb` change. No, we don't need to set both, and in fact `req.url.path` is read-only, so attempting to set it will fail.